app/controllers/quotes_controller.rb

```diff
@@ -1,14 +1,29 @@
 class QuotesController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Empty line should follow this (before a new def): https://github.com/StemboltHQ/ruby-style-guide#empty-lines-between-methods